src/components/rangeUsePlanPage/conditions/index.js

```diff
@@ -5,21 +5,32 @@ import { CONDITIONS } from '../../../constants/fields'
 import * as strings from '../../../constants/strings'
 import PermissionsField from '../../common/PermissionsField'
 import { TextArea } from 'formik-semantic-ui'
+import { useUser } from '../../../providers/UserProvider'
 
 const Conditions = ({ plan }) => {
-  const { conditions = '' } = plan || {}
+  const user = useUser()
+  const { conditions = '', proposedConditions = '' } = plan || {}
 
   return (
     <div className="rup__conditions">
       <div className="rup__popup-header">
         <div className="rup__content-title">{strings.CONDITIONS}</div>
-        <InfoTip header={strings.CONDITIONS} content={strings.CONDITIONS_TIP} />
       </div>
       <div className="rup__divider" style={{ marginBottom: 10 }} />
       <div className="rup__row">
         <div className="rup__cell-6" style={{ marginBottom: 40 }}>
+          <div className="rup__info-title">
+            <div className="rup__popup-header">
+              Decision Maker
```

Comment:
This should just be titled "conditions", otherwise I think agreement holders will be confused.